### PR TITLE
feature/viewModel-GoalsScreen

### PR DESCRIPTION
**Pull request by bsnh25**
1. Add view model for goal screen
2. Adjust goal screen

### CODE REVIEW

File: Thirtys/Screens/GoalsScreen.swift [hunk 1]
```diff
@@ -9,21 +9,21 @@ import SwiftUI
...
     
-    @Binding var goals_title: String
-    @Binding var goals_duration: String
     
     var body: some View {
         NavigationStack{
             ZStack{
                 Color.kBackground
                 VStack(spacing: 24){
                     VStack(spacing: 24){
```
Comment (RajeshTriadiNoftarizal) [inline Thirtys/Screens/GoalsScreen.swift:24]: vstack nya double, hapus satu

File: Thirtys/Screens/GoalsScreen.swift [hunk 2]
```diff
@@ -9,21 +9,21 @@ import SwiftUI
 
 struct GoalsScreen: View {
     
+    @EnvironmentObject private var gvm: GoalViewModel
```
Comment (ranof11) [inline Thirtys/Screens/GoalsScreen.swift]: change gvm to more descriptive variable

**Review by ranof11 — CHANGES_REQUESTED** [review]
ada di comments

**Review by bsnh25 — COMMENTED** [review]
done

**Review by ranof11 — APPROVED** [review]
lgtm

**Comment by RajeshTriadiNoftarizal** [comment]
merge to main branch

**Comment by ranof11** [comment]
merge to main branch

**Pull request merged**